DigitalOffice/Services/UserService/Broker/Consumers/UserConsumer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MassTransit;
+using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.InternalModels.BrokerInterface;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+
+namespace LT.DigitalOffice.UserService.Broker.Consumers
+{
+
+    public class UserConsumer : IConsumer<UserCredentialsInternalRequest>
+    {
+        private readonly IUserLoginCommand command;
+
+        public UserConsumer([FromServices] IUserLoginCommand command)
+        {
+            this.command = command;
+        }
+
+        public async Task Consume(ConsumeContext<UserCredentialsInternalRequest> context)
+        {
+            var request = context.Message;
+
+            var response = new
+            {
+                IsSuccess = true,
+                Errors = new List<string>(),
+                Body = command.Execute(request.Email)
+            };
+
```

Comment:
In the RabbitMQ context, it is incorrect to create new requests that are not RabbitMQ requests.
I mean that this variable is not used to send a request to the other service, but to the command.Execute.

To not confuse them, I suggest just type in "command.Execute(context.Message.Email)"